tests/phpunit/CRM/Mailing/BAO/MailingTest.php

```diff
@@ -397,6 +397,21 @@ public function testgetRecipientsEmailGroupIncludeExclude() {
     unset($expected[0], $expected[4], $expected[8]);
     $this->assertRecipientsCorrect($mailing['id'], $expected);
 
+    // Tear down: delete mailing, groups, contacts
+    $this->deleteMailing($mailing['id']);
+
+    // Create a New mailing, Testing contacts removed from smart group.
+    // In this case groupIDs6 will only pick up contacts[0] amd contacts[8] with it's
+    // criteria. However we are deliberly going to remove contactIds[8] from the group
```

Comment:
@seamuslee001 I think you mean "deliberately" - not "deliberly"

Reply:
or he was slurring....